tests/sha2.cc

```diff
@@ -83,6 +83,43 @@ TEST(ApiSuite, ApiTest)
 
     EXPECT_EQ(strncmp((char*)digest, (char*)expected_digest.data(), 32), 0);
   }
+
+  // Documentation.
+  // Lines after START and before END are used in documentation.
+  {
+    // ANCHOR(example streaming)
+    // This example uses SHA2-256.
+    //
```

Comment:
Remove?

Reply:
Can you say more?

Reply:
It's a useless/empty comment. Looks pretty weird.